src/Phoenix/PhoenixClient.ts

```diff
@@ -41,6 +52,65 @@ export class PhoenixClient {
       throw error;
     }
   }
+  public async setContainerHeartBeat(containerData: { forwardingId: string; dbAccountName: string }) {
```

Comment:
This is more like initiating the container heart beat, right? can we rename the function to reflect that? Set seems to indicate its setting a single property and not making periodic calls